schemas/org.gnome.feedreader.inoreader.gschema.xml

```diff
@@ -0,0 +1,25 @@
+<schemalist>
+	<schema id="org.gnome.feedreader.inoreader" path="/org/gnome/feedreader/inoreader/" gettext-domain="inoreader">
+	<key name="inoreader-api-token" type="s">
+		<default>""</default>
+		<summary>Acces key to Feedly-API</summary>
```

Comment:
The summary string is still `Acces key to Feedly-API` it should be `Acces key to InoReader API`